spec/honeycomb/integrations/redis_spec.rb

```diff
@@ -2054,7 +2054,17 @@
 
         let(:redis) do
           allow(connection).to receive(:read).and_return(slots, nodes, commands)
-          Redis.new(driver: driver, cluster: cluster, replica: true).tap do
+          Redis.new(
+            driver: driver, cluster: cluster, replica: true,
+          ).tap do |redis|
+            # https://github.com/redis/redis-rb/issues/1017
+            # establishes connections to all nodes in the cluster,
+            # otherwise extra READONLY commands are executed on replicas
+            if VERSION > Gem::Version.new("4.1.1")
+              redis.auth
+            else
+              redis.auth("")
```

Comment:
Oh, hm. 🤔 I assume it makes sense to pass a string (the password for the `AUTH` command) on all versions of Redis, rather than switch off of v4.1.1. That's what the other tests do: https://github.com/honeycombio/beeline-ruby/blob/adf2e8c3c27e308a94be8bd8d285b7ec92ed40e1/spec/honeycomb/integrations/redis_spec.rb#L185-L185 I just neglected to include it in the diff I commented with before. 😅